draft-scenarios/clash/arcane_artillery.tex

```diff
@@ -0,0 +1,155 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{Arcane Artillery}{\images/berserk.png}
```

Comment:
Berserk is already used, please pick something else from here: https://github.com/Heegu-sama/Homm3BG/tree/main/assets/spells